Vagrantfile

```diff
@@ -1,5 +1,7 @@
 # -*- mode: ruby -*- # vi: set ft=ruby :
 
+ENV['VAGRANT_NO_PARALLEL'] = 'yes'
```

Comment:
why ? this is debug?

Reply:
In my system, without that line Vagrant would intermittently fail while creating boxes.

Reply:
that is probably your system issue or box issue, that shouldn't affect changes here.